### PR TITLE
Foam enforcer tweaks

### PR DESCRIPTION
**Pull request by michaelyesok**
## What Does This PR Do

This PR gives the foam enforcer new sprites which make it more colourfull and more like a toy gun and a decs as the other one had a few typos and didnt fit in.
## Why It's Good For The Game

Make the foam enforcer actuelly look like a toy gun.
## Images of changes

New
image
image
OId
image
image

## Changelog
:cl:
imageadd: New foam enforcer sprites 
tweak: Tweaked foam enforcer desc
/:cl:

### CODE REVIEW

**Review by S34NW — APPROVED** [review]
Looks good

File: code/modules/projectiles/guns/projectile/toy.dm [hunk 1]
```diff
@@ -42,8 +42,8 @@
 	..()
 
 /obj/item/gun/projectile/automatic/toy/pistol/enforcer
-	name = "Foam Enforcer"
-	desc = "A foam shooting version of the Enforcer meant to be used for training new caddets who can't be trusted with rubber bullets."
+	name = "foam enforcer"
+	desc = "A toy inspired by the popular enforcer pistol. Ages 8 and up!"
```
Comment (dearmochi) [inline code/modules/projectiles/guns/projectile/toy.dm:46]:
```suggestion
	name = "foam Enforcer"
	desc = "A toy inspired by the popular Enforcer pistol. Ages 8 and up!"
```

**Comment by LiberatedWaveMan** [comment]
Looks pretty good to me, the sprites are absolutely better than the last ones, for sure. Description is solid as well.

**Comment by Mitchs98** [comment]
What pains me the most about the old foam enforcer is someone misspelled 'cadet' 😔

**Comment by DogsParadise** [comment]
I have approved these sprites as well.

Person in charge of this pr knows that.

**Review by dearmochi — APPROVED** [review]

**Comment by Carthusia** [comment]
The orange tip means its safe to shoot your friends with!

**Review by Kyep — APPROVED** [review]

**Review by SabreML — APPROVED** [review]
Looks genuinely amazing. 👍

**Review by variableundefined — APPROVED** [review]

**Pull request merged**